test/100-bridge-iptables.bats

```diff
@@ -635,41 +635,91 @@ EOF
+    run_netavark --file ${TESTSDIR}/testfiles/isolate3.json setup $(get_container_netns_path 2)
+
+    create_container_ns
+    run_netavark --file ${TESTSDIR}/testfiles/isolate4.json setup $(get_container_netns_path 3)
+
     # check iptables NETAVARK_ISOLATION_1 chain
     run_in_host_netns iptables -S NETAVARK_ISOLATION_1
-    assert "${lines[1]}" == "-A NETAVARK_ISOLATION_1 -i isolate2 ! -o isolate2 -j NETAVARK_ISOLATION_2"
-    assert "${lines[2]}" == "-A NETAVARK_ISOLATION_1 -i isolate1 ! -o isolate1 -j NETAVARK_ISOLATION_2"
+    assert "${lines[1]}" == "-A NETAVARK_ISOLATION_1 -i isolate4 ! -o isolate4 -j NETAVARK_ISOLATION_3"
+    assert "${lines[2]}" == "-A NETAVARK_ISOLATION_1 -i isolate3 ! -o isolate3 -j NETAVARK_ISOLATION_3"
+    assert "${lines[3]}" == "-A NETAVARK_ISOLATION_1 -i isolate2 ! -o isolate2 -j NETAVARK_ISOLATION_2"
+    assert "${lines[4]}" == "-A NETAVARK_ISOLATION_1 -i isolate1 ! -o isolate1 -j NETAVARK_ISOLATION_2"
 
     run_in_host_netns iptables -nvL FORWARD
     assert "${lines[2]}" =~ "NETAVARK_ISOLATION_1"
 
     # check iptables NETAVARK_ISOLATION_2 chain
     run_in_host_netns iptables -S NETAVARK_ISOLATION_2
-    assert "${lines[1]}" == "-A NETAVARK_ISOLATION_2 -o isolate2 -j DROP"
-    assert "${lines[2]}" == "-A NETAVARK_ISOLATION_2 -o isolate1 -j DROP"
+    assert "${lines[1]}" == "-A NETAVARK_ISOLATION_2 -o isolate4 -j DROP"
+    assert "${lines[2]}" == "-A NETAVARK_ISOLATION_2 -o isolate3 -j DROP"
+    assert "${lines[3]}" == "-A NETAVARK_ISOLATION_2 -o isolate2 -j DROP"
+    assert "${lines[4]}" == "-A NETAVARK_ISOLATION_2 -o isolate1 -j DROP"
+
+    # check iptables NETAVARK_ISOLATION_3 chain
+    run_in_host_netns iptables -S NETAVARK_ISOLATION_3
+    assert "${lines[1]}" == "-A NETAVARK_ISOLATION_3 -j NETAVARK_ISOLATION_2"
 
     # ping our own ip to make sure the ips work and there is no typo
     run_in_container_netns ping -w 1 -c 1 10.89.0.2
     run_in_container_netns 1 ping -w 1 -c 1 10.89.1.2
+    run_in_container_netns 2 ping -w 1 -c 1 10.89.2.2
+    run_in_container_netns 3 ping -w 1 -c 1 10.89.3.2
 
     # make sure the isolated network cannot reach the other network
     expected_rc=1 run_in_container_netns ping -w 1 -c 1 10.89.1.2
+    expected_rc=1 run_in_container_netns ping -w 1 -c 1 10.89.2.2
+    expected_rc=1 run_in_container_netns ping -w 1 -c 1 10.89.3.2
     expected_rc=1 run_in_container_netns 1 ping -w 1 -c 1 10.89.0.2
+    expected_rc=1 run_in_container_netns 1 ping -w 1 -c 1 10.89.2.2
+    expected_rc=1 run_in_container_netns 1 ping -w 1 -c 1 10.89.3.2
+    expected_rc=1 run_in_container_netns 2 ping -w 1 -c 1 10.89.0.2
+    expected_rc=1 run_in_container_netns 2 ping -w 1 -c 1 10.89.1.2
+    expected_rc=1 run_in_container_netns 2 ping -w 1 -c 1 10.89.3.2
+    expected_rc=1 run_in_container_netns 3 ping -w 1 -c 1 10.89.0.2
+    expected_rc=1 run_in_container_netns 3 ping -w 1 -c 1 10.89.1.2
+    expected_rc=1 run_in_container_netns 3 ping -w 1 -c 1 10.89.2.2
```

Comment:
Please add comments here what checks what, i.e from network x to y. If one thing here fails it is a nightmare to debug.